tests/test_modeling_tf_blenderbot.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2020 HuggingFace Inc. team.
+# Copyright 2021 The Facebook, Inc and The HuggingFace Inc. team. All rights reserved.
```

Comment:
As Sylvain likes to ask, did the Facebook Inc authors really contribute to this test file?